src/node/interfaces.cpp

```diff
@@ -841,10 +841,7 @@ class ChainImpl : public Chain
     }
     bool deleteRwSettings(const std::string& name, interfaces::SettingsAction action) override
     {
-        args().LockSettings([&](common::Settings& settings) {
-            settings.rw_settings.erase(name);
-        });
-        return action != interfaces::SettingsAction::WRITE || args().WriteSettingsFile();
+        return overwriteRwSetting(name, {}, action);
```

Comment:
In commit "chain: simplify `deleteRwSettings` code and improve it's doc" (84663291275248fd52da644b0c2566bbf9cc780b)

Might be good to say in commit message that this change is a refactoring and does not change behavior. Could have "refactor:" prefix in the title.

Reply:
I will do it when their is need to retouch.